src/ExtranetBundle/objects/WebModules/MTExtranetCore/MTExtranetCoreEndPoint.class.php

```diff
@@ -837,7 +837,7 @@ protected function PrepareSubmittedData(&$aData)
      *
      * @return void
      */
-    protected function PrepareSubmittedDataForUserAddress(&$aData)
+    protected function PrepareSubmittedDataForUserAddress($aData)
```

Comment:
Ich würde vermuten, dass hier das `&` durchaus gerechtfertigt ist.

Reply:
you are right